pytorch_pfn_extras/onnx/load.py

```diff
@@ -1,12 +1,11 @@
-# mypy: ignore-errors
-
 import onnx
 import json
 import warnings
 from pathlib import Path
+from typing import Any, IO
 
 
-def load_model(f, format=None, load_external_data=True):
+def load_model(f: IO, format: Any = None, load_external_data: bool = True) -> Any:
```

Comment:
Maybe https://github.com/onnx/onnx/blob/94fb49259018fc9ed95732bd7eb609bd518b0ee8/onnx/__init__.py#L108
```suggestion
def load_model(f: IO, format: Any = None, load_external_data: bool = True) -> onnx.ModelProto:
```

Reply:
Thanks!